src/channels/SignDidChannels/types.ts

```diff
@@ -1,18 +1,19 @@
+import type { Did, DidUrl, ICredential } from '@kiltprotocol/types';
+
 import { HexString } from '@polkadot/util/types';
-import { DidResourceUri, DidUri, ICredential } from '@kiltprotocol/sdk-js';
 
 import { DAppName } from '../../dApps/AccessChannels/DAppName';
 import { Origin } from '../../dApps/AccessChannels/Origin';
 
 export type SignDidInput = DAppName & {
   plaintext: string;
-  didUri?: DidUri;
+  didUri?: Did;
 };
 
 export type SignDidOriginInput = SignDidInput & Origin;
 
 export interface SignDidOutput {
   signature: HexString;
-  didKeyUri: DidResourceUri;
+  didKeyUri: DidUrl;
```

Comment:
Just a heads-up, mind that the new definition of `DidUrl` can include optional query parameters, so something like `did:kilt:4....?param=value#keyId` would now be a valid URL. They can of course simply be ignored, but I wanted to raise awareness that that's also possible now.